website_multi_theme/demo/pages.xml

```diff
@@ -23,11 +39,19 @@
         </t>
     </template>
 
-    <record id="demo_page_link" model="website.menu">
-        <field name="name">Multi website theme demo page</field>
-        <field name="url">/page/website_multi_theme.demo_page</field>
-        <field name="parent_id" ref="website.main_menu" />
-        <field name="sequence" type="int">50</field>
+    <record id="demo_page" model="website.page">
+        <field name="website_published">True</field>
+        <field name="url">/website_multi_theme</field>
+        <field name="view_id" ref="demo_page_view" />
+    </record>
+
+
+    <record id="menu_demo_page" model="website.menu">
+        <field name="page_id" ref="demo_page" />
+    </record>
+
+    <record id="menu_demo_page2" model="website.menu">
```

Comment:
Aren't you able to create these 2 menu records here instead of creating them at the top and re-editing them here at the bottom?

Reply:
I can't. Odoo source has the same approach. The reason in few computed fields that depends on each other. I will send details soon

Reply:
Well, I don't have the answer at the moment. When I did I got some problem and switched to a "splitting solution" and it began work. But now I tried to do it without splitting and it works too.

The splitting in odoo source may somehow related to splitting website.page and ir.ui.view creation for pages:

https://github.com/odoo/odoo/blob/4ecbacaf59576a22ff45615a5aa5c67244e4fb93/addons/website/data/website_data.xml#L34-L42

https://github.com/odoo/odoo/commit/8baabfb6be9e1db1af0828d705d07a1ec6f0029a#diff-d19576011e13311d9a4dd46879ebd724